addons/analytic/views/analytic_account_views.xml

```diff
@@ -29,7 +29,7 @@
                         <field name="analytic_distribution_ids" nolabel="1" widget="one2many_list"
                             attrs="{'invisible': [('active_analytic_distribution', '=', False)]}" groups="analytic.group_analytic_accounting">
                             <tree string="Analytic Distribution" editable="bottom">
-                                <field name="account_id"/>
+                                <field name="account_id" options="{'no_create': True}"/>
```

Comment:
this is an analytic account, if I'm not wrong... so we probably don't want to restrict its creation here